examples/plot_run_demo_solver.py

```diff
@@ -1,11 +1,22 @@
+"""
+=========
+Plot demo
+=========
+
+Plot demo.
```

Comment:
The docstring here is necessary to make `sphinx_gallery` happy. I put dummy text in here. But happy to change it to whatever you like.

Reply:
Fixed